sonaranalyzer-dotnet/src/SonarAnalyzer.CFG/ControlFlowGraph/AbstractControlFlowGraphBuilder.cs

```diff
@@ -113,20 +99,20 @@ private void ComputePredecessors()
 
         protected AbstractControlFlowGraphBuilder(SyntaxNode node, SemanticModel semanticModel)
         {
-            this.rootNode = node ?? throw new ArgumentNullException(nameof(node));
+            rootNode = node ?? throw new ArgumentNullException(nameof(node));
```

Comment:
A minor commit regarding harmonization of field names and usage of this was not backported to the security CFG. It would be nice to remove these changes to keep the "newest" version regarding this